Agile Essay Re-reference.tex

```diff
@@ -0,0 +1,68 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{How does agile methodology affect the workload on a team when in practise?}
+
+% Please do not change the subtitle
+\subtitle{COMP150 - Agile Development Practice}
+
+% Please put your student number in the author field
+\author{1608289}
+
+\begin{document}
+
+\maketitle
+
+\abstract{Proposal}
+
+The topic of my proposed essay is how forms of agile methodology affect the workload on a team when in practise in a workplace. In the recent past few years many companies and also development studios have been using scrum as their form of agile methodology. Throughout this essay I will go through both the pros and cons to putting scrum into practise and how it affects the workload on employees. Through doing so I can address many affects it will have on the quality of work, efficiency of work and also morale of a team during the scrum process. 

```

Comment:
You've left the abstract outside of the abstract command.

It should be like this:

```
\abstract{The topic of my proposed essay is how forms of agile methodology affect the workload on a team when in practise in a workplace. In the recent past few years many companies and also development studios have been using scrum as their form of agile methodology. Throughout this essay I will go through both the pros and cons to putting scrum into practise and how it affects the workload on employees. Through doing so I can address many affects it will have on the quality of work, efficiency of work and also morale of a team during the scrum process.}```